### DISCUSSION TITLE
[Question] Opacity of Spawned Entities

### DISCUSSION THREAD

**Discussion by YZhang001** [isaac-sim/IsaacLab#510] [Q&A]
Hello, I was following the tutorial on Orbit about the differential IK controller ([Link](https://isaac-orbit.github.io/orbit/source/tutorials/05_controllers/run_diff_ik.html))

For my application I need to visualize several non-physical entities in the scene, such as spheres on a table or markers. 

For the non-physical spheres, I tried to spawn them as follows in the `class TableTopSceneCfg(InteractiveSceneCfg)` (line 60 of `orbit/source/standalone/tutorials/05_controllers/run_diff_ik.py`): 
```
    sphere_1 = AssetBaseCfg(
            prim_path="{ENV_REGEX_NS}/Pose_Sphere_" + str(1),
            spawn=sim_utils.UsdFileCfg(usd_path=f"{ISAAC_NUCLEUS_DIR}/Props/Shapes/sphere.usd", 
                scale=(0.1, 0.1, 0.1),
            init_state=AssetBaseCfg.InitialStateCfg(pos=(0.4, -0.2, 0.0)))
```
and got spheres as shown in the picture, with a default color and default opacity: 
![image](https://github.com/NVIDIA-Omniverse/orbit/assets/75090909/5356ad48-8386-46da-8c1d-9ff4fed6f8c3)

_**My question is**_ whether there is a way for Orbit where I could spawn such a sphere with any color & opacity as needed, so that the sphere can be seen through and the scene/objects inside the sphere become visible. 

I found that it is possible to create a custom marker with customized opacity similar to the ones in `/orbit/source/extensions/omni.isaac.orbit/omni/isaac/orbit/markers/config/__init__.py` by setting the opacity of their `sim_utils.PreviewSurfaceCfg` config (`/orbit/source/extensions/omni.isaac.orbit/omni/isaac/orbit/sim/spawners/materials/visual_materials_cfg.py`); however, for the marker setting, the opacity doesn't make the markers transparent, but "only affects the surface's appearance during interactive rendering." 

As a further question, I saw from the Isaac Sim 2023.1.1 [release note](https://docs.omniverse.nvidia.com/isaacsim/latest/release_notes.html#id1) that Franka Research 3 (FR3) manipulator has been added as a new robot. I'd like to kindly ask whether FR3 will also be added to Orbit later, because I didn't find it under the {ISAAC_ORBIT_NUCLEUS_DIR}/Robots/ path. 

Thank you!

**Comment by Mayankm96** [comment]
Does setting the visual material as glass solves the problem for you?

Reference: https://isaac-orbit.github.io/orbit/source/api/orbit/omni.isaac.orbit.sim.spawners.html#omni.isaac.orbit.sim.spawners.materials.GlassMdlCfg
